### PR TITLE
Added custom server support. Added beta navidrome support (possibly h…

### PR DESCRIPTION
**Pull request by CarbonMac**
…acky) and updated the .env and README files

### CODE REVIEW

File: .env.example [hunk 1]
```diff
@@ -2,4 +2,9 @@ DISCORD_TOKEN=SOME_TOKEN_HERE
 POSTGRES_HOST=EXAMPLE_HOST
 POSTGRES_PASSWORD=EXAMPLE_PASSWORD
 POSTGRES_USER=EXAMPLE_USER
-POSTGRES_DB=EXAMPLE_DB
\ No newline at end of file
+POSTGRES_DB=EXAMPLE_DB
+#sub_url=subsonic.myawesomedomain.com
```
Comment (OpenSrcerer) [inline .env.example]: The format SCREAMING_SNAKE_CASE should be maintained for environment variables.

File: README.md [hunk 2]
```diff
@@ -5,7 +5,7 @@
   <a href="https://github.com/OpenSrcerer/supersonic/network">
     <img src="https://img.shields.io/github/forks/OpenSrcerer/supersonic?style=flat-square" width=86/>
   </a>
-  <a href="https://github.com/OpenSrcerer/thermostat/issues">
+  <a href="https://github.com/OpenSrcerer/supersonic/issues">
```
Comment (OpenSrcerer) [inline README.md:8]: Thanks, I missed this one when copying the ReadMe from my other repository!

File: src/main/kotlin/personal/opensrcerer/client/cache/SubsonicCache.kt [hunk 3]
```diff
@@ -2,15 +2,20 @@ package personal.opensrcerer.client.cache
 
 object SubsonicCache {
     private val cacheMap: MutableMap<String, SubsonicConfig>
+    private val url = System.getenv("sub_url") ?: "bonkersmusic.onthewifi.com"
```
Comment (OpenSrcerer) [inline src/main/kotlin/personal/opensrcerer/client/cache/SubsonicCache.kt]: SupersonicConstants.getVariable(String)

**Comment by OpenSrcerer** [comment]
Thank you for being the first contributor on this repository!
This commit is going to make this project much more friendly to users who don't know much about the project.
I plan to make process of adding servers simple, and all through Discord, but this will do just fine until we get to that point.

**Comment by CarbonMac** [comment]
Thank you for writing up something my lazy ass has been putting off! Fun little way for me to bounce over and see the Java side of the pond.

File: src/main/kotlin/personal/opensrcerer/client/cache/SubsonicCache.kt (hunk 3 above)
Comment (CarbonMac) [reply src/main/kotlin/personal/opensrcerer/client/cache/SubsonicCache.kt]: If I try to compile with this
`e: /buildapp/src/main/kotlin/personal/opensrcerer/client/cache/SubsonicCache.kt: (5, 23): Unresolved reference: SupersonicConstants`

Comment (CarbonMac) [reply src/main/kotlin/personal/opensrcerer/client/cache/SubsonicCache.kt]:
```    private val cacheMap: MutableMap<String, SubsonicConfig>
    private val url = SupersonicConstants.getVariable("SUB_URL") ?: "bonkersmusic.onthewifi.com"
    private val port = SupersonicConstants.getVariable("SUB_PORT").toIntOrNull() ?: 4040
    private val user = SupersonicConstants.getVariable("SUB_USER") ?: "Bonkers"
    private val pass = SupersonicConstants.getVariable("SUB_PASS") ?: "DNFWTF4201?!"
    private val version = SupersonicConstants.getVariable("SUB_VERSION") ?: "1.15.0"```

**Comment by OpenSrcerer** [comment]
> Thank you for writing up something my lazy ass has been putting off! Fun little way for me to bounce over and see the Java side of the pond.

It's a great learning experience for sure! The more you dabble in different languages, the more programming jokes you will find funny.

File: src/main/kotlin/personal/opensrcerer/client/cache/SubsonicCache.kt (hunk 3 above)
Comment (OpenSrcerer) [reply src/main/kotlin/personal/opensrcerer/client/cache/SubsonicCache.kt]: You need to add `import personal.opensrcerer.launch.SupersonicConstants` in the file imports.

**Comment by CarbonMac** [comment]
I think I have everything covered, The assertions were required due to null value stuff getting in the way

**Pull request merged**